### PR TITLE
Correct the suggested input instruction on the Activity Log filtering controls

### PR DESCRIPTION
**Pull request by TomScarboroughYMCA**
**Related Issue/Ticket:**

**PLEASE CHECK BASE BRANCH FOR YOUR PR**
**ONLY urgent and approved fixes for point release should go to master branch**

https://github.com/ymcatwincities/openy_gated_content/issues/119

## Steps to test:

- [ ] Log into any Virtual Y site as Site Owner or Administrator
- [ ] Navigate to Manage > Virtual Y > Logs
- [ ] Specify a Date Start and Date End
- [ ] Whereas before, the instruction text stated "Format: YYYY-MM-DD HH:MM:SS", now the instruction text states "Format: YYYY-MM-DD", which is consistent with what can be entered by the user without the ability to enter a space character.

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
  - [X] This PR  provides updates via `hook_update_N` or other means.
  - [ ] No updates are necessary for this change.
- Front end fixes should be tested against all of the Open Y Themes.
  - [X] Tested against Carnation
  - [X] Tested against Lily
  - [X] Tested against Rose
  - [ ] This change does not contain front-end fixes.
- [X] I have flagged this PR "Needs Review" or pinged the VY devs/QA
  team in Slack

### CODE REVIEW

**Review by hamrant — APPROVED** [review]
Code looks ok for me

**Comment by anpolimus** [comment]
retest

File: modules/openy_gc_log/openy_gc_log.install [hunk 1]
```diff
@@ -186,3 +186,16 @@ function openy_gc_log_update_8010() {
...
+
+/**
+ * Import updated logs view config.
+ */
+function openy_gc_log_update_8011() {
+  $config_dir = drupal_get_path('module', 'openy_gc_log');
+  $config_dir .= '/config/install/';
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
```
Comment (anpolimus) [inline modules/openy_gc_log/openy_gc_log.install:198]: @TomScarboroughYMCA why do you want to comment this code?

**Review by anpolimus — CHANGES_REQUESTED** [review]

File: modules/openy_gc_log/openy_gc_log.install (hunk 1 above)
Comment (TomScarboroughYMCA) [reply modules/openy_gc_log/openy_gc_log.install:198]: To be honest - I completely cut and pasted the _update blocks I found above this one, which had those comments. I don't have strong feelings on it.

Comment (froboy) [reply modules/openy_gc_log/openy_gc_log.install:168]: @anpolimus can you please clarify your question? The docblock is required as per coding standards, but is there another comment that should go away?

Comment (anpolimus) [reply modules/openy_gc_log/openy_gc_log.install:198]: @froboy, @TomScarboroughYMCA sorry. I've enabled dark mode at my Github and it was looking like you have commented code of the hook_update.
SOrry.
Everything is fine.

**Review by anpolimus — APPROVED** [review]

**Comment by AnastasiiaPys** [comment]
retest

**Comment by AnastasiiaPys** [comment]
Tested. Testing is passed.

**Pull request merged**